sdk/ai/Azure.AI.Agents.Persistent/samples/Sample19_PersistentAgents_VectorStoreFile_WithSteps.md

```diff
@@ -0,0 +1,283 @@
+# Sample file search using `VectorStoreFile` and agents in Azure.AI.Agents.Persistent.
+
+In this example we will create the local file, upload it and its file ID to new vector store using. Then we will drill down into run steps agent have ran through.
+1. First we need to create agent client and read the environment variables that will be used in the next steps.
```

Comment:
I would use Agent (capital A) everywhere. And it's "to create the Agent client" (add "the")

Reply:
Fixed.